README.md

```diff
@@ -609,6 +622,38 @@ Currently there exist three conversation history cache implementations:
 1. Redis cache
 1. Postgres cache
 
+Entries stored in cache have compound keys that consist of `user_id` and `conversation_id`. It is possible for one user to have multiple conversations and thus multiple `conversation_id` values at the same time. Global cache capacity can be specified. The capacity is measured as number of entries; entries size are ignored in this computation.
+
+#### In-memory cache
+
+In-memory cache is implemented as a queue with defined maximum capacity specified as number of entries that can be stored in a cache. That number is limit for all cache entries, not matter how many users are using LLM. When new entry is put into the cache and if the maximum capacity is reached, oldest entry is removed from the cache.
+
+#### Redis cache
+
+Entries are stored in Redis as dictionary. LRU policy can be specified that allows Redis to automatically remove oldest entries.
+
+#### Postgres cache
+
+Entries are stored in one Postgres table with following schema:
+
+```
+     Column      |            Type             | Nullable | Default | Storage  |
+-----------------+-----------------------------+----------+---------+----------+
+ user_id         | text                        | not null |         | extended |
+ conversation_id | text                        | not null |         | extended |
+ value           | bytea                       |          |         | extended |
+ updated_at      | timestamp without time zone |          |         | plain    |
+Indexes:
+    "cache_pkey" PRIMARY KEY, btree (user_id, conversation_id)
+    "cache_key_key" UNIQUE CONSTRAINT, btree (key)
+    "timestamps" btree (updated_at)
+Access method: heap
+```
+
+During new record insertion the maximum number of entries is checked and when the defined capacity is reached, oldest entry is deleted.
```

Comment:
During **a** new record (...), the oldest entry is deleted.

Reply:
fixed, thank you